pkg/client/scout/reporter.go

```diff
@@ -200,6 +203,22 @@ func (r *Reporter) Start(ctx context.Context) {
 	}()
 }
 
+func (r *Reporter) Close() {
+	// Send a zeroed bufEntry here instead of closing the buffer so that
+	// any stray Reports that arrive after the context is cancelled aren't
+	// sent on a closed channel
+	select {
+	case r.buffer <- bufEntry{}:
+	default:
+	}
+	// Wait for the done channel to close. Give up after 2 seconds (that
```

Comment:
this should probably read 3 seconds as does the code

Reply:
Indeed.

Reply:
I fix that in the next PR which unfortunately builds on this one.